Unity3D/Assets/RosSharp/Scripts/ImageManager.cs

```diff
@@ -0,0 +1,84 @@
+﻿using RosSharp.RosBridgeClient;
```

Comment:
As I have a lot of change requests in this file I uplad a new pull request to your fork for this file.
Let us then merge that pull request prior to this one.